### PR TITLE
Minor improvements to question 1; remove intro text from questions.

### PR DESCRIPTION
**Pull request by seanlip**

### CODE REVIEW

File: assets/questions/findMostCommonCharacter.js [hunk 1]
```diff
@@ -74,32 +72,81 @@ class AuxiliaryCode(object):
...
+      allowedOutputs: ['f'],
+      tag: 'strings with many spaces'
     }],
     buggyOutputTests: [{
       buggyFunctionName: 'AuxiliaryCode.lettersOnly',
       messages: [
         [
-          "Try running your code on '1600Amphitheatre'. ",
-          "Do you get the result you expected?"
+          'Note that the characters in the input strings may not always be ',
```
Comment (eyurko) [inline assets/questions/findMostCommonCharacter.js]: I'd drop the ellipsis and go for "always be numbers."

Comment (seanlip) [inline assets/questions/findMostCommonCharacter.js]: Er, did you mean "always be letters"?

Comment (eyurko) [inline assets/questions/findMostCommonCharacter.js]: yeah, that, whatever :P

Comment (seanlip) [inline assets/questions/findMostCommonCharacter.js]: haha :) done!

**Pull request merged**